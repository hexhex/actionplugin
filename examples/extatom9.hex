% parser test
parser0 :- &testZeroArity0[]().

% parser test
parser1 :- &testZeroArity0().

% parser test
parser2 :- &testZeroArity0[].

% parser test
parser3 :- &testZeroArity0.

% success test
success_ok :- &testZeroArity1.

% failure test
failure_ok :- not &testZeroArity0().

