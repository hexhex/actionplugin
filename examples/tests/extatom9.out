{failure_ok, success_ok}
